admin/create-dumps.sh

```diff
@@ -89,6 +90,14 @@ else
     exit
 fi
 
+# Lock cron, so it cannot be accidentally terminated.
+admin/cron_lock.py lock-cron create-dumps "Creating $DUMP_TYPE dump."
+
+# Trap should not be called before we lock cron to avoid wiping out an existing lock file
+trap on_exit EXIT
```

Comment:
To confirm what you mean by this comment - if `lock-cron create-dumps` fails (because the lock file already exists, and you want to quit the script), we shouldn't run `unlock-cron`, because that'd delete the lock file for the already running job, not this one that we just did?

Reply:
Correct.